internal/controller/streams.go

```diff
@@ -36,6 +36,8 @@ type streamClient struct {
 	client                    pb.KubernetesInfoServiceClient
 	disableNetworkFlowsCilium bool
 	falcoEventChan            chan string
+	flowCollector             pb.FlowCollector
+	flowCollectorWG           sync.WaitGroup
```

Comment:
I'm sorry, I gave you the wrong feedback.
We don't need synchronization / a WaitGroup. We're already doing things in the right order.
```suggestion
```